simulator/opendc-experiments/opendc-experiments-allocateam/src/main/kotlin/org/opendc/experiments/allocateam/experiment/monitor/RunMonitor.kt

```diff
@@ -59,19 +66,26 @@ public class RunMonitor(private val run: Run, private val clock: Clock) {
     }
 
     public fun reportJobStarted(event: WorkflowEvent.JobStarted) {
-        startTimesPerJob[event.job] = event.time
+        startTimesPerJob[event.jobState.job] = event.time
         this.startedJobs += 1
+        submissionTimesPerJob[event.jobState.job] = event.jobState.submittedAt
 //        logger.info { "Job ${event.job.uid} started" }
     }
 
     public fun reportJobSubmitted(event: WorkflowEvent.JobStarted) {
-        startTimesPerJob[event.job] = event.time
+        startTimesPerJob[event.jobState.job] = event.time
```

Comment:
The method title says that it will report that a job is submitted, but the `startTimesPerJob` attribute is being updated.

Reply:
`reportJobSubmitted` is actually not used, so I removed it